sandbox/tests/unit tests/inputs/test_projectfilereader_configurationblocks.appleseed

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project format_revision="27">
+<project format_revision="28">
```

Comment:
It would be nice to find a better solution than having to update this data file every time :)